service/orchestrator/metrics.json

```diff
@@ -1,128 +1,421 @@
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "OSLoggingOutput"
+    "id": "OSLoggingOutput",
+    "name": "OS Logging: Output",
+    "description": "This metric is used to assess to which service the OS logs are sent",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
+        ]
+      }
+    }
   },
   {
-    "id": "OSLoggingRetention"
+    "id": "OSLoggingRetention",
+    "name": "OS Logging: Retention",
+    "description": "This metric is used to assess the configured OS log retention (days)",
+    "scale": 3,
+    "range": {
+      "minMax": {
+        "min": 0,
+        "max": 365
+      }
+    }
   },
   {
-    "id": "ResourceInventory"
+    "id": "ResourceInventory",
+    "name": "Resource Inventory",
+    "description": "This metric is used to assess if an inventory is set up for resources",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "RoleBroadAssignments"
+    "id": "RoleBroadAssignments",
+    "name": "Role Broad Assignments",
+    "description": "This metric is used to assess the broad assignments of roles",
+    "scale": 3,
+    "range": {
+      "minMax": {
+        "min": 0,
+        "max": 1
+      }
+    }
   },
   {
-    "id": "RuntimeVersion"
+    "id": "RuntimeVersion",
+    "name": "Runtime Version",
+    "description": "This metric is used to assess the runtime version of a given programming language",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "*"
+        ]
+      }
+    }
   },
   {
-    "id": "SingleSignOnEnabled"
+    "id": "SingleSignOnEnabled",
+    "name": "Single Sign On Enabled",
+    "description": "This metric is used to assess if SSO is enabled",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          true,
+          false
+        ]
+      }
+    }
   },
   {
-    "id": "TLSVersion"
+    "id": "TLSVersion",
+    "name": "TLS Version",
+    "description": "This metric is used to assess the TLS version used for transfer data",
+    "scale": 1,
+    "range": {
+      "allowedValues": {
+        "values": [
+          "1.0",
+          "1.1",
+          "1.2",
+          "1.3"
+        ]
+      }
+    }
   },
   {
     "id": "TransportEncryptionAlgorithm",
     "name": "Transport Encryption: Protocol Version",
     "category": "CKM-02",
-    "description": "This metric describes, whether a up-to-date transport encryption protocol version is used",
+    "description": "This metric is used to assess if a up-to-date transport encryption protocol is used",
     "scale": 1,
     "range": {
       "order": {
         "values": [
-          "TLS 1.0",
-          "TLS 1.1",
-          "TLS 1.2",
-          "TLS 1.3"
+          "TLS"
```

Comment:
shouldn't this include the version numbers?

Reply:
TBH, I always stumble over our approach with the two metrics `TransportEncryptionAlgorithm` and `TLSVersion`. The former checks if we use an appropriate protocol, e.g. TLS but could also be another in theory. And the latter only argues about the TLS version. Or do you see each TLS version as its own EncryptionAlgorithm? If so, the `TLSVersion` metric is redundant .
I think either we do sth. like `TransportEncryptionAlgorithm` and `TransportEncryptionAlgorithmVersion`, or just `TLSXXX` metrics^^

Reply:
I feel the same way and would prefer to use only one metric like TLSVersion. :smile:

Reply:
Since there are no other suitable transport encryption algorithms besides TLS I think it’s not really practical to have a metric with just „TLS“. I would propose to merge them but with a TransportEncryption „prefix“